### ISSUE TITLE
libclang: crash detected in code completion

### ISSUE THREAD

**Issue by dabrahams** [Golevka/emacs-clang-complete-async#25]
I built this against llvm’s trunk:

libclang: crash detected in code completion
/Users/dave/bin/clang-complete: line 2: 13871 Segmentation fault: 11  DYLD_LIBRARY_PATH=/usr/local/llvm-trunk/lib ~/src/emacs-clang-complete-async/clang-complete "$*"

Process clang-complete exited abnormally with code 139

I’ll try building against clang-3.2 now

**Comment by yaruopooner** [comment]
same issue maybe.
crash by clang_reparseTranslationUnit.
http://clang-developers.42468.n3.nabble.com/clang-3-2-libclang-crash-detected-during-reparsing-td4029793.html

This bug has been fixed in version 3.3(svn trunk).
Clang3.3 is required if you want to try.

clang_reparseTranslationUnit use by ac-clang-update-cmdlineargs.

I use a temporary measure.

(defadvice ac-clang-update-cmdlineargs (around ac-clang-update-cmdlineargs-patch activate)
  (interactive)
  (ac-clang-shutdown-process)
  (ac-clang-launch-completion-process)
  )

**Comment by dabrahams** [comment]
Yes, things look good with clang3.3, thanks.  I guess your temporary measure is needed only if I am using an earlier clang?
